.tekton/gitops-operator-pull-request.yaml

```diff
@@ -18,6 +18,12 @@ metadata:
   namespace: rh-openshift-gitops-tenant
 spec:
   params:
+  - name: build-platforms
+    value:
+      - linux/x86_64
+      - linux/arm64
+      - linux/ppc64le
+      - linux/s390x
```

Comment:
```suggestion
```
Let's run PR builds only on amd64. I noticed long delays in setting up build env for non amd archs.

Reply:
Thanks for bringing this up, I was thinking of the same thing, It would be better we use amd64 for PRs.